### DISCUSSION TITLE
fastgpt 的 root 密码在 **前端** 怎么修改？

### DISCUSSION THREAD

**Discussion by piwawa** [labring/FastGPT#3450] [Q&A]
![image](https://github.com/user-attachments/assets/4c7e7d0a-98e1-43a5-b0b8-e2626b455eca)

这里居然修改密码都没有，放了一堆没用的东西

**Comment by c121914yu** [comment]
文档和 yml 文件注释看不懂嘞？https://doc.fastgpt.cn/docs/development/docker/#%E5%A6%82%E4%BD%95%E4%BF%AE%E6%94%B9%E5%AF%86%E7%A0%81

**Reply by piwawa** [reply]
> 文档和 yml 文件注释看不懂嘞？https://doc.fastgpt.cn/docs/development/docker/#%E5%A6%82%E4%BD%95%E4%BF%AE%E6%94%B9%E5%AF%86%E7%A0%81

我不说了 **前端** ？环境变量能改和前端有啥关系？

**Reply by piwawa** [reply]
> 文档和 yml 文件注释看不懂嘞？https://doc.fastgpt.cn/docs/development/docker/#%E5%A6%82%E4%BD%95%E4%BF%AE%E6%94%B9%E5%AF%86%E7%A0%81

怕你看不清，给标题也加了着重符号

**Reply by c121914yu** [reply]
> > 文档和 yml 文件注释看不懂嘞？https://doc.fastgpt.cn/docs/development/docker/#%E5%A6%82%E4%BD%95%E4%BF%AE%E6%94%B9%E5%AF%86%E7%A0%81
> 
> 怕你看不清，给标题也加了着重符号

文档都写有咋改密码了，和哪里改有啥关系？
